### ISSUE TITLE
Shadowroot chest has no breaking particles when just mining it

### ISSUE THREAD

**Issue by Terenx** [Cryptic-Mushroom/Midnight#40]
when trying to mine the chest without breaking it completely, the particles aren't there

**Comment by gegy1000** [comment]
Is this the latest version?

**Comment by Terenx** [comment]
Yes, if you smash the chest completely you see the particles, but in the process of mining it, it does emit any particles

**Comment by Terenx** [comment]
You fixed one issue but it seems to have caused another

**Comment by gegy1000** [comment]
That.. doesn't make any sense. Will look into it.

**Comment by Terenx** [comment]
Lemme break it down, you have to go through the process of mining the item before it breaks and you can pick up the block or item the block drops when broken, when mining a block it usually gives off some breaking particles, and when fully broken it explodes into these particles and drops the block/item

**Comment by Terenx** [comment]
The particles from the fully breaking of the block appear, but not the initial mining particles

**Comment by gegy1000** [comment]
Yep, but particles are handled the same in both cases.

**Comment by Terenx** [comment]
Then idk whats causing it, I will look into it again in the morning to see if I can still duplicate the bug and maybe screenshot it
